src/targets/browser/browserLauncher.ts

```diff
@@ -144,6 +144,18 @@ export class BrowserLauncher implements ILauncher {
     );
   }
 
+  prepareWebViewLaunch(params: IChromeLaunchConfiguration) {
+    // port 2015 is carried over from vscode-edge-debug2
+    params.port = params.port || 2015;
```

Comment:
This should not be required for new Edge. 2015 was used because with Legacy Edge there is only one debug server for all Edge processes.  That is not the case with Chrome/MsEdge.

Reply:
Done.